### PR TITLE
Added a Core idExpiryDays configuration option.

### PR DESCRIPTION
**Pull request by icio**
With this, users can control the endurance of cookies so that they satisfy their privacy policy's guidelines.

### CODE REVIEW

File: lib/core.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { promiseRetry, bind, uuid } from './util';
  * @typedef {object} CoreConfig
  * @prop {string} [key] The API key ("publishable_key_..." or "pk_...") used to authenticate with the Ravelin API.
  * @prop {string} [api] A string of the format "https://api.ravelin.net" which forms the base of API requests.
+ * @prop {number} [idTTLDays=365] The max lifetime of a deviceId, in days.
```
Comment (icio) [inline lib/core.js]: Maybe `idExpiryDays` would be clearer?

File: README.md [hunk 2]
```diff
@@ -165,6 +165,11 @@ var rav = new Ravelin({
      * Content-Security-Policy then add the api to the connect-src directive.
      */
     // api: 'https://live.ravelin.click/',
+    /**
+     * @prop {number} [idTTLDays] The number of days that a device ID will live.
+     * Defaults to 365 in accordance with the GCPR's ePrivacy Directive.
```
Comment (grantstephens) [inline README.md]: GDPR?
```suggestion
     * Defaults to 365 in accordance with the GDPR's ePrivacy Directive.
```

**Review by grantstephens — APPROVED** [review]
LGTM.
Don't know if we need to worry about people setting it to a negative number, but I suppose that is their problem at the end of the day?

**Comment by icio** [comment]
Survey says `idExpiryDays`! I'll change that before merging.

**Comment by sweeney** [comment]
Still refers to `idTTLDays` in the readme

**Comment by icio** [comment]
> Still refers to `idTTLDays` in the readme

Thanks! Turns out I still had a "*.js" file filter enabled when I searched/replaced.

**Pull request merged**